sonar-python-plugin/src/test/java/org/sonar/plugins/python/cpd/PythonCpdAnalyzerTest.java

```diff
@@ -76,11 +76,11 @@ public void code_chunks_2() throws Exception {
       "deffoo():pass",
       "classbar(object):pass",
       "deffoo2(x,y,z,):",
-      "pass",
+      "pass\n",
```

Comment:
add maybe a comment to explicitly say that `new_line` is there to mark the DEDENT on the next line.